### PR TITLE
✨: Add feature modules for qa app

### PR DESCRIPTION
**Pull request by creature-water-valley**
## ✅ What's done

- [x] Add feature modules for qa app
- [x] Add qa app navigation

## ⏸ What's not done

Just added the minimum functionality to transition to each screen of the qa app.

---

## Tests

- [x] Must be able to transition to each screen.

以下のコマンドのいずれかをこのプルリクエストのコメントとして投稿すると、
Azure Pipeline上でSantokuAppをビルドしてDeployGateへアップロードできます。
4種全てのビルドバリアントを対象にする場合はdeploy-all、
特定のビルドバリアントだけを対象にする場合はdeploy-ビルドバリアント名のコマンドを利用してください。

- /azp run deploy-all
- /azp run deploy-devSantokuAppDebugAdvanced
- /azp run deploy-devSantokuAppReleaseInHouse
- /azp run deploy-santokuAppDebugAdvanced
- /azp run deploy-santokuAppReleaseInHouse

## Devices

- [ ] 動作確認に利用したデバイスにチェックをつけてください
- [ ] iOS
  - [x] シミュレータ (iPhone 13/iOS 15)
  - [ ] 実機 (iPhone 8/iOS 14)
- [ ] Android
  - [x] エミュレータ (Pixel 4a/Android 12)
  - [ ] 実機 (Pixel 3a/Android 11)

## Other (messages to reviewers, concerns, etc.)

none

### CODE REVIEW

File: example-app/SantokuApp/src/apps/app/navigators/RootStackNav.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ const useRootStackNavigator = (initialData: AppInitialData) => {
               name="DemoStackNav"
               component={DemoStackNav}
               options={{
```
Comment (in-a-paddy) [inline example-app/SantokuApp/src/apps/app/navigators/RootStackNav.tsx:61]: It might be better to remove `options`.

Comment (creature-water-valley) [reply example-app/SantokuApp/src/apps/app/navigators/RootStackNav.tsx]: Fixed
50a8e76c9b12e3d48fd90eb55ba4f1299b6c0a32

**Review by in-a-paddy — APPROVED** [review]

**Comment by kozake** [comment]
At this point, I feel that the functional modules are divided is too detailed.

- qa-event
- qa-home
- qa-post
- qa-question

Do you envision many more features being extended to each functional module in the future?

**Comment by creature-water-valley** [comment]
> At this point, I feel that the functional modules are divided is too detailed.
> Do you envision many more features being extended to each functional module in the future?


As noted in "What's not done," this PR only adds the minimum functionality to transition to each screen of the qa app.

Therefore, in order to create the Figma as designed, other elements will be added as they are added.

Or 「do we assume that we will expand the functionality even more than Figma's content?」 Is that the meaning of this?

If so, there will be no expansion of functionality.

I don't think the division is too detailed, but I don't have any major objection to combining them into one.

What do @in-a-paddy  think?

**Comment by in-a-paddy** [comment]
I think it's not too devided. This app has following features.
* Home
* Question
* Event

And users can post a question or event from the same screen, So post feature is devided from event and question.

**Comment by kozake** [comment]
> this PR only adds the minimum functionality to transition to each screen of the qa app.

Understood. So, let's keep this.

**Review by kozake — APPROVED** [review]

**Pull request merged**